subsys/mgmt/mcumgr/Kconfig

```diff
@@ -123,6 +123,14 @@ config FS_MGMT_PATH_SIZE
 	  and download commands.
 endif
 
+# Must use the dummy backend for the shell so that mcumgr can capture output
```

Comment:
Move that comment into help for the option.

Reply:
That makes sense. Will do. Thanks!